### PR TITLE
Fix/3580 exact match for mention me

### PR DESCRIPTION
**Pull request by raphj**
#3580 

## Checkpoints

**For developers**

- [X] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [X] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [X] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/mention.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ export const addClassToMentionsOfUser = (rawContent, username, userClassName = M
 
   const releventMentions = [username, ...GROUP_MENTION_TRANSLATION_LIST]
```
Comment (grignards) [inline frontend_lib/src/mention.js:96]: s/relevent/relevant/:

```suggestion
  const relevantMentions = [username, ...GROUP_MENTION_TRANSLATION_LIST]
```

**Review by grignards — CHANGES_REQUESTED** [review]

**Review by gbonaspetti — APPROVED** [review]

File: frontend_lib/src/mention.js (hunk 1 above)
Comment (raphj) [reply frontend_lib/src/mention.js:96]: One day, I'll know how to write relevant. ^^

Comment (raphj) [reply frontend_lib/src/mention.js]: fixed

**Review by grignards — APPROVED** [review]

**Comment by gbonaspetti** [comment]
@raphj During the tests I noticed two things in this PR:
1. With I tried to send a comment with `@allllllllllllllllllllllllllllllllllll` I have a generic error "Error while saving the comment" even if backend sends a more precise error. (same for html-document)
2. If I write `@alllllllllllllllllllllllllllllllllllllllll` and I click `Enter` the text disappears, this doesn't happen with `space` key
enter

**Comment by asf-algoo** [comment]
> With I tried to send a comment with @allllllllllllllllllllllllllllllllllll I have a generic error "Error while saving the comment" even if backend sends a more precise error. (same for html-document)

Indeed. I didn't intend to fix this in this PR, but I can.

> If I write @alllllllllllllllllllllllllllllllllllllllll and I click Enter the text disappears, this doesn't happen with space key
I am currently working on this in another PR.

**Comment by raphj** [comment]
@gbonaspetti both of these issues are not relevant to this PR, which is about matching a mention after saving.

I can handle your first point in the PR I am working on, and probably fixed 2 in it (I will double check).
   
Are you ok with merging this fix?

**Comment by raphj** [comment]
I confirm both points are fixed in #3611

**Comment by PhilippeAccorsi** [comment]
Test en cours.

**Review by gbonaspetti — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Verified.

**Pull request merged**